pkgs/by-name/li/limine/package.nix

```diff
@@ -0,0 +1,62 @@
+# Builds limine with all available features.
+
+{
+  # Helpers
+  stdenv
+, fetchurl
+, lib
+, # Dependencies
+  llvmPackages
+, mtools
+, nasm
+}:
+
+let
+  version = "7.1.0";
+in
+# The output of the derivation is a tool to create bootable images using Limine
+# as bootloader for various platforms and corresponding binary and helper files.
+stdenv.mkDerivation {
+  inherit version;
+  pname = "limine";
+  # We don't use the Git source but the release tarball, as the source has a
+  # `./bootstrap` script performing network access to download resources.
+  # Packaging that in Nix is very cumbersome.
+  src = fetchurl {
+    url = "https://github.com/limine-bootloader/limine/releases/download/v${version}/limine-${version}.tar.gz";
+    sha256 = "sha256-oaQA461MGVuG7aJri2HyYb6Pu8BXOWTF3Wq/SputQSw=";
+  };
+
+  nativeBuildInputs = [
+    llvmPackages.bintools
+    # gcc is used for the host tool, while clang is used for the bootloader.
+    llvmPackages.clang
+    llvmPackages.lld
+    mtools
+    nasm
+  ];
+
+  configureFlags = [ "--enable-all" ];
+
+  enableParallelBuilding = true;
+  # the bootloader build takes care of the relevant flags by itself
+  NIX_HARDENING_ENABLE = "";
```

Comment:
Is this necessary? You know, you probably don't want to touch hardening flags that affect the normal `limine` executable that runs on the host... The bootloader itself uses `CFLAGS_FOR_TARGET` and `LDFLAGS_FOR_TARGET` and, as you said, it takes care of most flags necessities by itself.

Reply:
This was more a preliminary measurement. Probably unneeded. Removed it.